<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BigFileMessageClass</class>
 <widget class="QDialog" name="BigFileMessageClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>770</width>
    <height>195</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>BigFileMessage</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>3</number>
   </property>
   <property name="topMargin">
    <number>3</number>
   </property>
   <property name="rightMargin">
    <number>3</number>
   </property>
   <property name="bottomMargin">
    <number>6</number>
   </property>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Open Mode</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="2">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Big Text File(&lt; 8G) Read only open, load in blocks, and turn pages manually.</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QRadioButton" name="hexMode">
        <property name="text">
         <string>Hex Bin</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QRadioButton" name="superBigTextMode">
        <property name="text">
         <string>Super Big Text Edit</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Binary Open,load in blocks, and turn pages manually.</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QRadioButton" name="textMode">
        <property name="text">
         <string>Text Mode</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Open directly in text mode.May be slow, Need wait.</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QRadioButton" name="bigTextMode">
        <property name="text">
         <string>Big Text</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Super big Text File(&gt; 8G bits) Read only open, load in blocks, and turn pages manually.</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="okBt">
       <property name="text">
        <string>Ok</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelBt">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>okBt</sender>
   <signal>clicked()</signal>
   <receiver>BigFileMessageClass</receiver>
   <slot>slot_okBt()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>359</x>
     <y>168</y>
    </hint>
    <hint type="destinationlabel">
     <x>284</x>
     <y>193</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelBt</sender>
   <signal>clicked()</signal>
   <receiver>BigFileMessageClass</receiver>
   <slot>slot_cancelBt()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>448</x>
     <y>176</y>
    </hint>
    <hint type="destinationlabel">
     <x>446</x>
     <y>196</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>slot_okBt()</slot>
  <slot>slot_cancelBt()</slot>
 </slots>
</ui>
